client/src/pages/component/homeBody/Homebody.css

```diff
@@ -140,7 +141,8 @@
 			justify-content: center;
 			align-items: center;
 			width: 100%;
-			height: 250px;
+			height: 350px;
```

Comment:
you probably shouldn't be heighting in pixels

Reply:
thank you @gregdyke i will consider that and start changing it